pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -412,6 +412,58 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: PersistentVolumeControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// PersistentVolumeBinder.volumeController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("get", "update", "create", "delete"),
```

Comment:
The provisioning pieces (which were declined for 3.1) will require Create and Delete of PVs.